content/20.zksync-protocol/15.zksyncos/01.bootloader.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Bootloader
+description: Learn about Bootloader, the entrypoint of the system
+---
+
+The bootloader is the component responsible for implementing the general blockchain protocol. Roughly, this means:
+
+1. Initializing the system.
+2. Reading the block context from oracle.
+3. Reading and parsing the first transaction.
+4. Validating the transaction.
+5. Executing the transaction.
+6. Saving the transaction result.
+7. Repeat from 3 until there are no more transactions.
+8. Finalizing the block.
+
+This component is, as the name suggest, the entrypoint of the system.
+The function  [`run_prepared`](https://github.com/matter-labs/zk_ee/blob/main/basic_bootloader/src/bootloader/mod.rs) implements this
+top-level main loop.
+
+### Configuration
+
+The bootloader can be configured with the following parameters
+(found in the [`BasicBootloaderExecutionConfig`](https://github.com/matter-labs/zk_ee/blob/main/basic_bootloader/src/bootloader/mod.rs) struct):
+
+- `ONLY_SIMULATE`: skips the [validation](/zksync-protocol/zksyncos/tx-processing#validation) step
+when processing a transaction. Used for call simulation in the node.
+- `IS_PROVING_ENVIRONMENT`: to skip some checks during sequencing.
+- `SPECIAL_ADDRESS_SPACE_BOUND`: the range of address (starting from 0) where system contracts can be deployed.
+- `AA_ENABLED`: whether native account abstraction is enabled.
+
+In addition, the `basic_bootloader` crate has the following compilation flags:
+
+- `code_in_kernel_space`: to enable normal contract execution for addresses in the range `[0, SPECIAL_ADDRESS_SPACE_BOUND]`.
+- `transfers_to_kernel_space`: to enable token transfers to addresses in the range `[0, SPECIAL_ADDRESS_SPACE_BOUND]`. Note: the bootloader itself
+has a special formal address (0x8001) that is always allowed to receive token transfers. This is used to collect fees.
+- `charge_priority_fee`: to enable charging for the EIP-1559 tip (priority fee) on top of the base fee.
+- `evm-compatibility`: enables all the previous flags, needed for the EVM test suite.
+
+### Account Abstraction
+
+Transaction validation and execution is done through an
+abstract [account model](/zksync-protocol/zksyncos/tx-processing#account-models). This is because BoojumOS supports
```

Comment:
We might want to remove this section, as we're not gonna support AA. Also, a "BoojumOS" slipped through